tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,11 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
+    ###Assertions I added###
+    assert result == False
```

Comment:
We could assert the length of the inventory is the same as it was before the function call? Would we have to save the length to variable before the function call in order to do this?

Reply:
Nice, we should check the length. Since we're passing the hardcoded inventory `["a", "b", "c"]` above we know the length should be 3, so we could use 3 directly in the assert's comparison. 

Even if the length is the same, that doesn't guarantee the individual elements of the list are unchanged. To confirm the inventory contents had not been altered we could assert the individual items that should be in the list are there:
```
assert "a" in vendor.inventory
```

Reply:
Thank you! Makes sense.